src/tools/live-utils/device-cache.ts

```diff
@@ -0,0 +1,32 @@
+import fs from "fs";
```

Comment:
do not create another file for caching, use the same one, try to make it common for both live and app live.